source/basic.tex

```diff
@@ -401,7 +401,7 @@
 type \tcode{T} (Clause~\ref{conv}, \ref{expr.type.conv},
 \ref{expr.dynamic.cast}, \ref{expr.static.cast}, \ref{expr.cast}), or
 \item an expression that is not a null pointer constant, and has type
-other than \cv{} \tcode{void*}, is converted to the type pointer to \tcode{T}
+other than \cv{}~\tcode{void*}, is converted to the type pointer to \tcode{T}
 or reference to \tcode{T} using a standard conversion
```

Comment:
In general (not a comment on this change), instead of "cv void *", I'd like to see phrasing such as "pointer to cv void" (in double quotes). The latter is the "official" type language introduced in 8.3, and we should use it, in particular when we do not-quite-C++ things such as \cv with it.